incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -185,25 +214,28 @@ func TestPropagateConflict(t *testing.T) {
 				{Group: "", Resource: "secrets", Mode: "Propagate"}}
 			c := &api.HNCConfiguration{Spec: api.HNCConfigurationSpec{Resources: configs}}
 			c.Name = api.HNCConfigSingleton
-			// Create a forest with "a" as the parent and "b" and "c" as the children.
-			f := foresttest.Create("-aa")
+			f := foresttest.Create(tc.forest)
 			config := &HNCConfig{
 				translator: fakeGRTranslator{},
 				Forest:     f,
 				Log:        zap.Logger(false),
 			}
 
 			// Add source objects to the forest.
-			inst := &unstructured.Unstructured{}
-			inst.SetGroupVersionKind(schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Secret"})
-			inst.SetName("my-creds")
-			for _, c := range tc.inNamespaces {
+			for c, annot := range tc.inNsesWithAnnot {
```

Comment:
Note that this conflicts with the `c` defined on line 215. It'll still compile (because a `for` loop introduces a new scope) but it's confusing - you should change one of them.

Reply:
done